src/core/api/retention.go

```diff
@@ -151,6 +151,10 @@ func (r *RetentionAPI) CreateRetention() {
 		r.SendBadRequestError(err)
 		return
 	}
+	if len(p.Rules) > 15 {
+		r.SendBadRequestError(errors.New("there can be at most 15 rules"))
```

Comment:
This error message `"there can be at most 15 rules"` seems not very smooth.

“at most, only 15 rules are allowed”?